src/renderers/webxr/WebXRManager.js

```diff
@@ -182,9 +183,10 @@ class WebXRManager extends EventDispatcher {
 
 		}
 
-		this.setFramebufferScaleFactor = function ( value ) {
+		this.setFramebufferScaleFactor = function ( value, limitWithNativeScaleFactor = false ) {
```

Comment:
I think in this function, you can check for "XRWebGLLayer.getNativeFramebufferScaleFactor" and if value is bigger, set it to that.
Otherwise, I'm unsure if this code is needed..

Reply:
The function unfortunately depends on `session` which may not exist at the time this function is called.

Reply:
Bizarre. 
With that info, this looks ok. Is there documentation that needs to be updated?

Reply:
Added :)

Reply:
Let's use this opportunity and change:
```js
this.setFramebufferScaleFactor = function ( value, limitWithNativeScaleFactor = false ) {
```
```js
this.setFramebufferScaleFactor = function ( factor, limited = false ) {
```
The method name is already descriptive and so I think the parameter names can be shortened.

Reply:
Done!